warrior/Warrior

```diff
@@ -48,6 +49,77 @@ def update_jira_by_id(jiraproj, jiraid, exec_dir, status):
     else:
         print_info("jiraid not provided, will not update jira issue")
 
+def avg_time_calculate(abs_filepath, default_repo):
+    """ To print the Estimated Runtime for the Project/Suite/Testcase is
+    calculated using the previous junit file runtime values """
+
+    fname = abs_filepath.split('/')[-1].split('.')[0]
+    junit_fname = fname + "_junit.xml"
+    file_type = Utils.xml_Utils.getRoot(abs_filepath).tag
+    total = count = 0
+
+    if not default_repo.has_key('ow_resultdir'):
+        result_path = Utils.xml_Utils.getChildTextbyParentTag(abs_filepath,
+                                                               'Details',
+							       'Resultsdir')
+
+        if result_path is None or result_path is False:
+            file_path = abs_filepath.split('warriorframework/')[0]
+            exec_filepath = file_path + \
+                          "warriorframework/warrior/Warriorspace/Execution"
+
+        else:
+            file_path = abs_filepath.split(fname)[0]
+            exec_filepath = file_path + "Execution"
+
+        if file_type == "Project" or file_type == "TestSuite":
+            junit_file = exec_filepath + "/**/" + junit_fname
+            if os.path.exists(exec_filepath):
+                for filename in glob.iglob(junit_file):
+                    fd = open (filename, 'r')
```

Comment:
Junit is an XML file, use xml_util methods to parse it rather than treating it as normal text file.